### ISSUE TITLE
How to train Fully-supervised model?

### ISSUE THREAD

**Issue by Haerxu** [CVMI-Lab/PLA#21]
Hi,

I want to train a fully-supervised baseline ("Fully-sup." in table 1). Are the data augmentation, learning rate and optimizer the same as PLA?

**Comment by Dingry** [comment]
Yes. The hyper-params are the same. You can use this config to train a fully-sup model.  https://github.com/CVMI-Lab/PLA/blob/main/tools/cfgs/scannet_models/spconv_clip_adamw.yaml
